dask_glm/utils.py

```diff
@@ -164,6 +164,11 @@ def make_y(X, beta=np.array([1.5, -3]), chunks=2):
     y = da.random.random(z0.shape, chunks=z0.chunks) < sigmoid(z0)
     return y
 
+def cupy_make_y(X, beta=cupy.array([1.5, -3])):
+    n, p = X.shape
+    z0 = X.dot(beta)
+    y = cupy.random.random(z0.shape) < sigmoid(z0)
+    return y
```

Comment:
We probably can't have this library `import cupy` at the top level, otherwise users will get an ImportError if they don't have cupy installed.

Reply:
We might move this into the `dask_glm/tests/cupy` module and import from there.

Reply:
> Also, I'm surprised that there weren't any code changes. All I'm seeing here is testing code (which all looks quite nice). Did you have to modify any code to get things to work?

There were a few code changes, in the first commit , mostly regarding `*_like()` calls and such. Did you see that?

https://github.com/dask/dask-glm/pull/75/commits/e39aafd289193df56cb7e254a4aa174d3dd2dcff

Reply:
Yes, I was planning to cleanup this cupy imports later, but I hadn't yet thought of how I should do it, your idea sounds good, I will do that.

Reply:
Ah indeed, I must have accidentally been looking only at the most recent commit.